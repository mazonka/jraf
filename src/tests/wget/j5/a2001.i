command=jw login a@b.com * : logout f82d3bb17612458c