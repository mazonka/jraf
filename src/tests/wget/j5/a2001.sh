#!/bin/sh

cwd=`sh execdir.sh`

lg=$cwd/wroot/.jraf.sys/login/

echo ""
echo "DIR login =="
ls -1 $lg
echo "DIR ========"

wr=$cwd/wroot
rm -rf $wr



