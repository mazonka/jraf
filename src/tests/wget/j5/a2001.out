OK http://127.0.0.1:16000/jraf? : OK
DIR login ==
DIR ========
